src/libPMacc/PMaccConfig.cmake

```diff
@@ -29,8 +29,7 @@
 ###############################################################################
 # PMacc
 ###############################################################################
-cmake_minimum_required(VERSION 3.3)
-project("PMacc")
+cmake_minimum_required(VERSION 3.1)
```

Comment:
See above. Same version as PIConGPU and the `project` is just plain wrong here. This is a `*Config.cmake` file should not contain that but simply provide paths for `FindModule`